topics/statistics/tutorials/RNN/slides.html

```diff
@@ -0,0 +1,223 @@
+---
+layout: tutorial_slides
+logo: GTN
+video: true
```

Comment:
if you would like to create automated text-to-speech video for these slides, please add suitable speaker notes to every slide ([instructions](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/slides-with-video/tutorial.html)). If not, please remove this line

Reply:
Will remove it.